ansible/files/steps/batch.sh

```diff
@@ -89,10 +89,11 @@ else
     fi
     if [ $EXTENSION = "json" ]; then
         # Executes Zeppelin notebook
+        ZEPPELIN_PATH=/mnt/anaconda2/envs/zeppelin/bin
```

Comment:
I'm fine with this but an alternative could be to use the base path:

`ZEPPELIN_PATH=/mnt/anaconda2/envs/zeppelin`

and append `/bin` in the calls:

`${ZEPPELIN_PATH}/bin/zeppelin-execute ...`

Whichever you prefer.